src/main.rs

```diff
@@ -157,14 +144,6 @@ fn main() -> Result<(), Error> {
         }
     });
 
-    // Cache all of the components.
-    let component_watch = std::thread::spawn(move || loop {
-        if let Err(res) = reflector.poll() {
-            error!("Component polling error: {:?}", res);
-        };
-    });
-    info!("Watcher is running");
```

Comment:
I think this log line could be left here

Reply:
changed it to "ApplicationConfiguration watcher is running"